lib/driver.js

```diff
@@ -43,6 +43,8 @@ const DEFAULT_SETTINGS = {
   mjpegServerScreenshotQuality: 25,
   mjpegServerFramerate: 10,
   screenshotQuality: 1,
+  checkForModalDialogs: true,
```

Comment:
if this is Boolean variable then I would rather rename it to `shouldCheckModalDialogs` or `isModalDisalogsCheckEnabled`